test/components/portal.spec.tsx

```diff
@@ -1,36 +1,38 @@
 import * as React from 'react';
 import * as ReactDOM from 'react-dom';
-import {ClientRenderer, expect, selectDom, waitFor} from 'test-drive-react';
+import {ClientRenderer, expect, waitFor} from 'test-drive-react';
 import {Portal} from '../../src';
-
-const portalId = 'PORTAL';
+import {PortalTestDriver} from '../../test-kit/components/portal-driver';
 
 describe('<Portal />', function() {
     const clientRenderer = new ClientRenderer();
-    const bodySelect = selectDom(document.body);
 
     afterEach(function() {clientRenderer.cleanup(); });
 
     it('displays the portal and renders its children', async function() {
-        clientRenderer.render(
+        const {result} = clientRenderer.render(
             <Portal>
                 <span data-automation-id="SPAN">Portal Body</span>
             </Portal>);
 
+        const driver = new PortalTestDriver(result as Portal);
+
         await waitFor(() => {
-            expect(bodySelect(portalId)).to.be.present();
-            expect(bodySelect(portalId, 'SPAN')).to.be.present();
+            expect(driver.portal).to.be.present();
+            expect(driver.content[0]).to.be.present();
         });
     });
 
     it('applies supplied styles to the popup and updates them if changed', async function() {
-        const {container} = clientRenderer.render(
+        const {container, result} = clientRenderer.render(
             <Portal style={{position: 'absolute'}}>
                 <span data-automation-id="SPAN">Portal Body</span>
             </Portal>
         );
 
-        await waitFor(() => expect(bodySelect(portalId)).to.have.nested.property('style.position', 'absolute'));
+        const driver = new PortalTestDriver(result as Portal);
+
+        await waitFor(() => expect(driver.portal).to.have.nested.property('style.position', 'absolute'));
```

Comment:
add `style` getter to the driver and then match `property`

Reply:
after playing with the idea i think this is still more convenient to do than returning the whole style from the getter. besides, always possible to do `driver.portal.style.position`